requirements.txt

```diff
@@ -37,4 +37,4 @@ types-mock>=0.1.1
 types-setuptools>=57.0.0
 types-requests
 types-setuptools>=57.0.0
-black>=22.0,<23.0
+black==22.3.0
```

Comment:
Why do we pin to 22.3.0 now?

Reply:
The point here is more to have the same thing across the board, so this is reflecting the actual CI and pre-commit checks that were currently running.

The actual pin that we want could be up for discussion (also moving to the newest release), but is a separate question from syncing this better for the future.

Reply:
> The point here is more to have the same thing across the board, so this is reflecting the actual CI and pre-commit checks that were currently running.

Yes, I understood this. I was just curious about why this particular version instead of the previous range, but it's not central.